.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pybind11"]
+	path = pybind11
+	url = https://github.com/pybind/pybind11.git
```

Comment:
Is there a way to pin the version of pybind? Otherwise, this module will break as soon as there are breaking changes introduced into pybind11.

Reply:
Oh interesting, last time I added the submodule to another package it pinned it to a particular commit hash. I can do that